pkg/storaged/storage.css

```diff
@@ -723,3 +723,11 @@ td.storage-action {
 .sidepanel-row .spinner {
     display: inline-block;
 }
+
+.panel-heading {
+    position: static;
```

Comment:
Static is the default position for elements. Is this overriding something?

Reply:
This overrides our own pk/lib/table.css:

```
/* Fix panel heading alignment & mobile layout */

.panel-heading {
    align-items: center;
    background: #f5f5f5;
    display: flex;
    flex-wrap: wrap;
    /* (28px small size widget height) + (0.5rem * 2) */
    min-height: calc(28px + 1rem);
    padding: 0.5rem 0.5rem 0.5rem 1rem;
    position: relative;
    z-index: 100;
}
```

The `position: relative` makes it so that the dropdown menu would show behind the "Drives" panel:

![image](https://user-images.githubusercontent.com/3228183/68465242-c07c5380-021a-11ea-9a32-4eae5e39c612.png)

Reply:
Oh! Well then.

I assume the `position: relative` is because we're using floats* and want to make sure they don't escape.

_* **ugh!** Floats shouldn't be used for layout. I've gotten rid of a bunch, but we're still using a lot of floats and `pull-right`s (which use floats)._

Why shouldn't floats be used for layout? It's a hack. They're fragile. Also, in order to use them, you have to mess with document order (put the rightmost thing first) and this breaks accessibility (among other things).

But meanwhile, we do have some lingering floats and they need to be contained, so it needs to still be there for now, I guess.

_**TL;DR**: OK, you make a very good point and we do need `position: static` here. :wink:_